src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,238 @@
+//        System.out.println("Noted. I've removed this task:");
+//        System.out.println("  " + deletedTask);
+//        System.out.println("Now you have " + remainingTasks + " tasks in the list.");
+//        System.out.println("_____________________________");
+//    }
+//
+//    /**
+//     * Displays a message for an invalid task index.
+//     */
+//    public void showInvalidTaskIndex() {
+//        System.out.println("Invalid task index. Please provide a valid task index.");
+//    }
+//
+//    /**
+//     * Displays a message indicating a task has been marked as done.
+//     *
+//     * @param markedTask The marked task.
+//     */
+//    public void showTaskMarked(Task markedTask) {
+//        System.out.println("Nice! I've marked this task as done:");
+//        System.out.println("  " + markedTask);
+//        System.out.println("_____________________________");
+//    }
+//
+//    /**
+//     * Displays a message indicating a task has been marked as not done.
+//     *
+//     * @param unmarkedTask The unmarked task.
+//     */
+//    public void showTaskUnmarked(Task unmarkedTask) {
+//        System.out.println("OK, I've marked this task as not done yet:");
+//        System.out.println("  " + unmarkedTask);
+//        System.out.println("_____________________________");
+//    }
+//
+//    public void showMatchingTasks(List<Task> matchingTasks) {
+//        if (matchingTasks.isEmpty()) {
+//            System.out.println("No matching tasks found.");
+//        } else {
+//            System.out.println("Here are the matching tasks in your list:");
+//            for (int i = 0; i < matchingTasks.size(); i++) {
+//                System.out.println((i + 1) + "." + matchingTasks.get(i));
+//            }
+//        }
+//        System.out.println("_____________________________");
+//    }
+//
+//    /**
+//     * Displays a goodbye message.
+//     */
+//    public void showGoodbye() {
+//        System.out.println("Goodbye. Hope to see you again soon!");
+//        System.out.println("_____________________________");
+//    }
+//
+//    public String getResponseAndClear() {
+//        String response = responseBuilder.toString();
+//        responseBuilder.setLength(0); // Clear the builder after retrieving the response
+//        return response;
+//    }
+//
+//}
+package duke;
+
+import java.util.List;
+
+public class Ui {
+    private StringBuilder responseBuilder = new StringBuilder();
+
+    // Use this method to add messages to the responseBuilder with a newline
+    private void addMessage(String message) {
+        responseBuilder.append(message).append("\n");
+    }
+
+    public void showWelcome() {
+        addMessage("Hello! I'm Aether!");
+        addMessage("What can I do for you?");
+    }
+
+    public void showLoadingError() {
+        addMessage("Error loading tasks from file.");
+    }
+
+    public void showSeparator() {
+        addMessage("_____________________________");
```

Comment:
Consider using a constant for the separator line to ensure consistency across different methods.

Example:
private static final String SEPARATOR_LINE = "_____________________________";

Doing so, you won't have to copy/paste the line over and over again and can just
addMessage(SEPARATOR_LINE);